src/components/CardList.js

```diff
@@ -2,23 +2,24 @@ import React from 'react';
 import Card from './Card';
 
 
-const CardList = ({robots}) => {
-	return(
-		<div>
-			{
-			robots.map((user,i) => {
-			return (	
-				<Card 
-					key={i} 
-					id = {robots[i].id} 
```

Comment:
The use of index as the key is an [anti-pattern](https://reactjs.org/docs/lists-and-keys.html#keys)